### PR TITLE
Attribute no etchosts

### LINKED ISSUE #193: Add an attribute to disable entries automatic injections in `/etc/hosts`
**Is your feature request related to a problem? Please describe.**
`pot` populates `/etc/hosts` with all `pot`s reachable from the same bridge.
However, in very dynamic environments, this features is quite useless

**Describe the feature you'd like to have**
An attribute to disable the automatic entries injection in `/etc/hosts`

**Describe potential alternatives or workaround you've considered (if any)**
A local script that rewrite `/etc/hosts` at boot

### PR DESCRIPTION
**Pull request by pizzamig**
fix #193

### CODE REVIEW

File: share/zsh/site-functions/_pot [hunk 1]
```diff
@@ -27,7 +27,7 @@ _pot_flavours() {
 }
 
 _pot_attributes() {
-	_values "pot attributes" 'start-at-boot' 'early-start-at-boot' 'persistent' 'no-rc-script' 'prunable' 'localhost-tunnel' 'enforce_statfs' 'mount' 'fdescfs' 'linprocfs' 'nullfs' 'procfs' 'tmpfs' 'zfs' 'children'
+	_values "pot attributes" 'start-at-boot' 'early-start-at-boot' 'persistent' 'no-rc-script' 'prunable' 'localhost-tunnel' 'no-tmps' 'no-etc-hosts' 'enforce_statfs' 'mount' 'fdescfs' 'linprocfs' 'nullfs' 'procfs' 'tmpfs' 'zfs' 'children'
```
Comment (grembo) [inline share/zsh/site-functions/_pot:30]: no-tmps => no-tmpfs

File: share/pot/update-config.sh [hunk 2]
```diff
@@ -81,6 +81,10 @@ _update_one_pot()
 		_debug "pot.attr.no-tmpfs=NO"
 		echo "pot.attr.no-tmpfs=NO" >> "$_conf"
 	fi
+	if [ -z "$(_get_conf_var "$_pname" "pot.attr.no-tmpfs")" ]; then
```
Comment (grembo) [inline share/pot/update-config.sh:84]: Should be "pot.attr.no-etc-hosts"

**Review by grembo — CHANGES_REQUESTED** [review]

File: share/zsh/site-functions/_pot (hunk 1 above)
Comment (grembo) [reply share/zsh/site-functions/_pot:30]: Related: Is enforce_statfs something that should be changed to enforce-statfs at some point to match other attributes?

Comment (pizzamig) [reply share/zsh/site-functions/_pot]: nice catch.
`enforce_statfs` is inherited from `jail`, that's why it uses the `_`.
I don't have a strong opinion for or against `-` and `_`
Adopting only one separator would probably show a higher quality

File: share/pot/update-config.sh (hunk 2 above)
Comment (pizzamig) [reply share/pot/update-config.sh]: nice catch as well

**Review by grembo — APPROVED** [review]

**Pull request merged**